tests/unit/src/main/scala/tests/TestingServer.scala

```diff
@@ -1339,7 +1339,16 @@ final class TestingServer(
           PrettyPrintTree(
             label(uri),
```

Comment:
it fails actually here, and it failed again :/

Reply:
Right! Changed it. It's super weird it fails, labels are produced for exactly the same keys, which is weird this happens.

Reply:
So it seems the label exists, but it's still not being found in the map :/

Reply:
it is in the map according to logs!!! WTF

Reply:
Trying with stripping BOM

Reply:
nope 😭

Reply:
Sooo it seems again, it was `c:` instead of `C:` - why Windows? Why!?